dist/videojs-markers.js

```diff
@@ -98,6 +98,9 @@
             marker.div.on('click', function(e) {
                var key = $(this).data('marker-index');
                player.currentTime(markers[key].time);
+
+               if (marker.onMarkerClicked)
+                  marker.onMarkerClicked();
```

Comment:
You don't need to directly edit dist files. They should be generated from "grunt compile" (see grunt file for detail)